### PR TITLE
Fix - Tissue column in histopath landing heatmap

### PR DESCRIPTION
**Pull request by francisco-ebi**
Avoid triggering heatmap click handler by adding and specifying a class in the html structure and handler.

### CODE REVIEW

File: web/src/main/webapp/WEB-INF/views/histopathLandingPage.jsp [hunk 1]
```diff
@@ -210,7 +210,7 @@
             <tbody>
             <c:forEach var="arow" items="${rows}" varStatus="status">
                 <tr>
-                    <td>${geneSymbols[status.index]}</td>
+                    <td class="cell">${geneSymbols[status.index]}</td>
```
Comment (francisco-ebi) [inline web/src/main/webapp/WEB-INF/views/histopathLandingPage.jsp:213]: To preserve legacy behaviour (although I'm not sure if linking to a page like this: https://www.mousephenotype.org/data/histopath/1500009L16Rik?anatomy="Gene" is correct)

**Review by rwilson-ebi — APPROVED** [review]
Seems a sensible approach. I also don't think the anatomy parameter is used but leave it for now.

File: web/src/main/webapp/WEB-INF/views/histopathLandingPage.jsp (hunk 1 above)
Comment (ficolo) [reply web/src/main/webapp/WEB-INF/views/histopathLandingPage.jsp:213]: That seems to be working fine. We need to review this whole story when we re-implement the histopath pages in the new website.

**Review by ficolo — APPROVED** [review]
Tested it locally and works as expected.

**Pull request merged**